modules/ROOT/pages/_partials/task-view-governed-apis-for-rulesets.adoc

```diff
@@ -0,0 +1,27 @@
+// Used in docs-exchange/asset-details.adoc and docs-api-governance-documentation/view-conformance-status-in-exchange.adoc
+
+In a ruleset's *Governed APIs* page, you can view the list of centrally-governed APIs to which the ruleset applies. 
+
+Developers can use the *Governed APIs* list to help them improve the quality of the rulesets and APIs they develop as follows: 
+
+* When their APIs fail a ruleset, they can see which other APIs are successfully conforming to the ruleset so that they can fix their APIs.
+* When they are discovering available rulesets to use for their API, they can see which APIs are conformant to a specific ruleset to evaluate how helpful it is.
+
+API consumers can use the *Governed APIs* list when they need to create an API service or application that is conformant to a certain set of standards or rules. They can:
+
+* Ensure there is a ruleset in Exchange that has the rules they need.
+* See which APIs are conformant to that ruleset so that they can make sure their application is consuming only the conformant APIs.
+
+To view the governed API list for a ruleset:
+
+. Select a ruleset asset that is used in an API Governance profile. 
+. Select *Governed APIs* in the navigation bar to see the following information for each API listed:
+
+* Whether the API is governed by multiple centralized rulesets and by how many  
+* Whether the API passed the ruleset
```

Comment:
What does "passed" mean in this context? Passed to something or conforming to the ruleset?

Reply:
It is a fine point, but the words passed and failed are used for the rules in a single ruleset, but an API cannot be in conformant status unless it passes all rulesets in a profile. This is elsewhere in the doc, so shouldn't be repeated here.